test/e2e/multikueue/e2e_test.go

```diff
@@ -255,7 +255,8 @@ var _ = ginkgo.Describe("MultiKueue", func() {
 						Parallelism: 2,
 						Completions: 2,
 						Image:       "gcr.io/k8s-staging-perf-tests/sleep:v0.1.0",
-						Args:        []string{"1ms"},
+						// Give it the time to be observed Active in the live status update step.
+						Args: []string{"5s"},
```

Comment:
Can it be less? Maybe 3s?

Reply:
Yeah, I suggested 5s for consistency with other places. Ideally, I would like not to have 10min timeout, and once the checks are done, then finish the job, but this is ticketed under https://github.com/kubernetes-sigs/kueue/issues/1449

Reply:
sg